components/ipfs/ipfs_component_cleaner.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/ipfs/ipfs_component_cleaner.h"
+
+#include "base/files/file_util.h"
+#include "base/path_service.h"
+#include "base/task/thread_pool.h"
+#include "build/build_config.h"
+#include "chrome/common/chrome_paths.h"
```

Comment:
Also should not be in `components`. It's ok I think to fix in a followup PR

Reply:
Created follow up issue: https://github.com/brave/brave-browser/issues/39624